examples/js/effects/VREffect.js

```diff
@@ -11,14 +11,18 @@
 
 THREE.VREffect = function ( renderer, onError ) {
 
+	var scope = this;
+
 	var vrHMD;
 	var isDeprecatedAPI = false;
 	var eyeTranslationL = new THREE.Vector3();
 	var eyeTranslationR = new THREE.Vector3();
 	var renderRectL, renderRectR;
 	var eyeFOVL, eyeFOVR;
 
-	function gotVRDevices( devices ) {
+	function gotVRDevices ( devices ) {
+
+		scope.displays = devices;
```

Comment:
I had that originally; can change, for sure. well, actually the v1.0 API calls for `navigator.getVRDisplays()` instead of `navigator.getVRDevices()`, so I'd suggest the former.